airflow/contrib/operators/qubole_operator.py

```diff
@@ -95,7 +95,7 @@ class QuboleOperator(BaseOperator):
     """
 
     template_fields = ('query', 'script_location', 'sub_command', 'script', 'files', 'archives', 'program', 'cmdline', 'sql', 'where_clause', 'extract_query', 'boundary_query', 'macros', 'tags', 'name')
-    template_ext = ('.hql', '.sql', '.sh', '.bash', '.pig')
+    template_ext = ('.txt')
```

Comment:
All this is doing is disabling templating for your specific use case (ends in .hql). See my note in https://issues.apache.org/jira/browse/AIRFLOW-119 about this.

Rather than changing templating to use .txt, which will never work if anyone sets a .txt file, I think we should just remove templating support from the remote files.